docs/openapi/auth-schemas/auth.yaml

```diff
@@ -0,0 +1,80 @@
+info:
```

Comment:
Can we add some patter validation to the properties?
For example authorizationToken and fiscal_code. Check the front-end to get the right regular expression.

Reply:
try to add /^[A-Za-z0-9\-._~+/]+$/ for sessionToken and authorizationToken